python/ray/tune/execution/experiment_state.py

```diff
@@ -201,20 +201,23 @@ def wait_for_sync():
             if (
                 time_since_last_sync is not None
                 and time_since_last_sync < self._excessive_sync_threshold
+                and self._should_force_sync_up
```

Comment:
Just double checking my understanding, is this to only warn when the flow is `Checkpoint -> Checkpoint`, as opposed to `Checkpoint -> Experiment End`?

Reply:
Yes exactly